build.gradle

```diff
@@ -4,7 +4,8 @@ apply plugin: 'com.vanniktech.android.junit.jacoco'
 
 buildscript {
     ext.versions = [
-        testRules: '0.5',
+        espresso: '3.0.0',
```

Comment:
Concerning screengrab: Something set a different version, so I redefinied it here. Why does that happen?